src/VisualStudio/Core/Test.Next/Services/SolutionServiceTests.cs

```diff
@@ -446,7 +447,9 @@ public async Task TestRemoteWorkspace()
         await Verify(remoteWorkspace, currentSolution, remoteSolution3);
 
         // move to new solution backward
-        var solutionInfo2 = await assetProvider.CreateSolutionInfoAsync(await solution1.CompilationState.GetChecksumAsync(CancellationToken.None), CancellationToken.None);
+        var solutionInfo2 = await assetProvider.CreateSolutionInfoAsync(
+            await solution1.CompilationState.GetChecksumAsync(CancellationToken.None),
+            remoteWorkspace.Services.GetService<IAnalyzerAssemblyLoaderProvider>(), CancellationToken.None);
```

Comment:
>CancellationToken.None [](http://example.com/codeflow?start=84&length=22)

nit: stylistic preference for me would be to put the third param on it's own line too

Reply:
done.